lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -92,6 +102,10 @@ check_text_not_a_link:
   element:
     selector:
       xpath: //*[contains(@class,'co-resource-item__resource-name') and text()='<text>' and not(@href)]
+check_text_on_page:
+  element:
+    selector:
+      xpath: //*[contains(text(), '<text>')]
```

Comment:
Can it be replaced with `check_page_contains` or `check_page_match`?

Reply:
replaced with check_page_contains